projects/sd-jwt-js.md

```diff
@@ -1,80 +1,97 @@
+  - [Credo-ts](https://github.com/openwallet-foundation/credo-ts)
+  - [Credhub](https://github.com/openwallet-foundation-labs/credhub)
+  - [pex](https://github.com/Sphereon-Opensource/PEX)
+  - [ssi-kit](https://github.com/Sphereon-Opensource/SSI-SDK)
+- Demonstrate a substantial ongoing flow of commits and merged contributions.
+  - 12 [Contributors](https://github.com/openwallet-foundation-labs/sd-jwt-js/graphs/contributors) since project inception
+  - [178 PRs merged](https://github.com/openwallet-foundation-labs/sd-jwt-js/pulls?q=is%3Apr+is%3Aclosed)
+  - [Total 1.2M+ NPM Downloads](https://npm-stats-ecru.vercel.app/)
+- Demonstrate that the current level of community participation is sufficient to meet the goals outlined in the growth plan and roadmap.
+- Since these metrics can vary significantly depending on the type, scope and size of a project, the TAC has final judgment over the level of activity that is adequate to meet these criteria.
+- Demonstrates how this project differs from existing projects in the Growth and Impact stages.
 
 # Project Description
 
-This is the reference implmentation of [IETF SD-JWT specification](https://datatracker.ietf.org/doc/draft-ietf-oauth-selective-disclosure-jwt/) in Javascript.
+This is the reference implementation of [IETF SD-JWT specification](https://datatracker.ietf.org/doc/draft-ietf-oauth-selective-disclosure-jwt/), [IETF SD-JWT VC specification](https://datatracker.ietf.org/doc/draft-ietf-oauth-sd-jwt-vc/) and [Token Status List](https://datatracker.ietf.org/doc/draft-ietf-oauth-status-list/) in Javascript/Typescript.
 
-Hopae, a founding member of OpenWallet Foundation, is building wallet module in Javascript and need this project as a core component.
+Hopae, a founding member of OpenWallet Foundation, is building wallet module in Javascript/Typescript and need this project as a core component.
 
 # Alignment with the OpenWallet Foundation Mission
 
 This library is for the Selective Disclosure capability which is one of the important building blocks of digital identity wallets.
 
-# Current Code of Conduct
+# Code of Conduct
 
-none
+[OpenWallet Foundation code of conduct](https://tac.openwallet.foundation/governance/code-of-conduct/)
 
 # TAC Sponsor
 
-none
+@aceshim Ace Shim
 
 # Project License
 
 Apache 2.0
 
 # Source Control
 
-none
+https://github.com/openwallet-foundation-labs/sd-jwt-js
 
 # Issue Tracker
 
-none
+https://github.com/openwallet-foundation-labs/sd-jwt-js/issues
 
 # External Dependencies
 
-TBD
+- js-base64
 
 # Release Methodology
 
-TBD
+Project will be released on NPM. I have `@sd-jwt` organization on NPM for this project. Package name will be like `@sd-jwt/core`.
 
 # Initial Maintainers
 
 Ace Shim ([Github](https://github.com/pensivej))
-Gavin Kim ([Github](https://github.com/zustkeeper))
 Lukas Han ([Github](https://github.com/lukasjhan))
+Mirko Mollik ([Github](https://github.com/cre8))
```

Comment:
I forgot to add brend. https://github.com/berendsliedrecht
I'll add him after TAC meeting